contracts/contracts/harvest/OETHHarvesterSimple.sol

```diff
@@ -55,8 +69,13 @@ contract OETHHarvesterSimple is Strategizable {
             // Cache balance
             uint256 balance = IERC20(rewardTokens[i]).balanceOf(address(this));
             if (balance > 0) {
+                // Determine receiver
+                address receiver = rewardTokens[i] == WETH
```

Comment:
nit: Usage of `rewardTokens[i]` and `rewardTokens.length` can be extracted into their own variables. But since `rewardTokens` is local variable, it should be fine as it is.

Reply:
Added in this commit: https://github.com/OriginProtocol/origin-dollar/pull/2363/commits/fd5da5ed7dc7d97d5af88997089d1e7637930030.